### PR TITLE
Fixing auto fetch over due tasks

### PR DESCRIPTION
**Pull request by fakhruddinkw**
fixing auto over due fetch not working on prod

### CODE REVIEW

File: tests/unit/utils/taskOverDueDiscordMembers.test.ts [hunk 1]
```diff
@@ -27,11 +27,11 @@ describe("taskOverDueDiscordMembers()", () => {
...
       .mockRejectedValueOnce(new Error("Network error"));
 
-    const result = await taskOverDueDiscordMembers();
-
-    expect(result).toEqual(
-      "Oops! We have encountered an internal Server Error"
-    );
+    try {
+      await taskOverDueDiscordMembers();
+    } catch (error: any) {
```
Comment (iamitprakash) [inline tests/unit/utils/taskOverDueDiscordMembers.test.ts:32]: can't we log error anywhere

Comment (fakhruddinkw) [reply tests/unit/utils/taskOverDueDiscordMembers.test.ts:32]: > can't we log error anywhere

In discord-slash, only logging we have available is console.log statements

File: src/utils/taskOverDueDiscordMembers.ts [hunk 2]
```diff
@@ -20,6 +20,7 @@ export const taskOverDueDiscordMembers = async (): Promise<
 
     return assigneeIds;
   } catch (e) {
-    return errors.INTERNAL_SERVER_ERROR;
+    console.log(e);
+    throw e;
```
Comment (Pratiyushkumar) [inline src/utils/taskOverDueDiscordMembers.ts:24]: NIT - why console.log and why not console. error?
in my opining console.error will be much suited option here

**Review by Pratiyushkumar — COMMENTED** [review]
A demo and test coverage should also be included

**Comment by fakhruddinkw** [comment]
> A demo and test coverage should also be included

This is not a new functionality just fixing old one, thats why didnt include stats here.

**Comment by Pratiyushkumar** [comment]
> > A demo and test coverage should also be included
> 
> This is not a new functionality just fixing old one, thats why didnt include stats here.

Since test file changes are seen and hence stats are important also demo is important to include for reviewer to verify if the changes have gotten fixed or not.

**Review by prakashchoudhary07 — APPROVED** [review]

**Comment by fakhruddinkw** [comment]
> > > A demo and test coverage should also be included
> > 
> > 
> > This is not a new functionality just fixing old one, thats why didnt include stats here.
> 
> Since test file changes are seen and hence stats are important also demo is important to include for reviewer to verify if the changes have gotten fixed or not.

If the fix is for prod how would a dev know it is fixed or not if it is not merged on prod????

File: tests/unit/utils/taskOverDueDiscordMembers.test.ts (hunk 1 above)
Comment (fakhruddinkw) [reply tests/unit/utils/taskOverDueDiscordMembers.test.ts:32]: The logging is there, only that it will be logged on CF instead of our local in this case

**Comment by iamitprakash** [comment]
> > A demo and test coverage should also be included
> 
> This is not a new functionality just fixing old one, thats why didnt include stats here.

this is not accepted excuse

**Comment by fakhruddinkw** [comment]
> > > A demo and test coverage should also be included
> > 
> > 
> > This is not a new functionality just fixing old one, thats why didnt include stats here.
> 
> this is not accepted excuse

I am sorry, i was not making any excuses, i just gave my reason.

**Pull request merged**